stubs/model.blade.php

```diff
@@ -2,6 +2,9 @@
 
 use Illuminate\Database\Eloquent\Model;
 use RonasIT\Support\Traits\ModelTrait;
+@if(in_array('Carbon',$anotationProperties) || in_array('Carbon|null',$anotationProperties))
```

Comment:
let's use https://laravel.com/docs/12.x/helpers#method-array-hasany helper for it